RELEASING.md

```diff
@@ -0,0 +1,36 @@
+# Releasing Guide
+
+AX releasing workflow is semi-guided by the CI logic comprised of [GitHub actions scripts](.github/workflows/), [makefile](./Makefile), and [cosmos-release tool](./rust/release/).
+
+## General Release Steps
+
+> - Note: Explanations for the text in bold can be found in the [glossary section](#glossary).
+
+1. A release starts when a **changelog commit** is created.
+1. The previous step triggers the creation of a **Release PR**. This PR will initially fail because the version validation detects that a version bump is required.
+1. _[Manually]_ With a developer's assistance, each impacted **product's** version is bumped correctly. These product's version can be found in each of the corresponding cargo.toml or package.json file.
+1. _[Manually]_ the **Release PR** is merged when its build and validation chain is successful.
+1. After **Release PR** is merged to the `master` branch, a publication process will run; it publishes all **products** listed on the previously merged release.
+1. _[Manually]_ A developer publishes `rust` crates that have been changed.
+
+## Glossary:
+
+- **`Actyx` Product**: or simply product, refers to publishable artifacts from this repository (`ax`, `ax-core`, `node-manager`, etc).
+- **changelog commit**: a conventional commit with a valid product designation that will trigger a release process and will be included in the release changelog.
+- **versions file**: is a special file that lists supposedly released versions of **`Actyx` products**. This file is located [here](./versions).
+- **Release PR**: An automatically generated Pull Request for release purposes. It contains a changelog and the appending of the **version file**.
+- **Databank**: refers to the set of storage rules and the communication protocol inside `ax-core`. Previously this was a standalone binary simply called `Actyx`. A **Databank** has a version number to co-identify whether it has a compatible communication protocol with a peer **Databank**.
```

Comment:
```suggestion
- **databank version**: refers to the set of storage rules, communication protocols, and other behaviours inside `ax-core`.
  This is necessary because the Rust crate version of `ax-core` is dictated by semantic versioning, which regards only the exposed data types and function signatures, i.e. an orthogonal set of concerns.
    The databank version continues to denote what was previously the version of the `Actyx` binary (when `ax` was only its remote management CLI).
```